src/main/java/tech/pegasys/pantheon/plugin/services/PantheonConfiguration.java

```diff
@@ -23,4 +25,12 @@
    * @return location of the storage in the file system of the client.
    */
   Path getStoragePath();
+
+  /**
+   * Url of the enclave Pantheon is connected to if privacy is enabled
```

Comment:
```suggestion
   * Url of the enclave that stores private transaction data.
```

I'm not sure this is exactly right but trying to give a bit more flavour of what the enclave is actually used for.